lexico/word.py

```diff
@@ -175,7 +176,9 @@ def get_translations(word):
         return list()
 
     def stringify(self):
-
+        
+        spinner = Halo(text='LOADING', spinner='dot', color='yellow')
```

Comment:
Move this to the `cli.py` so that we can separate the application and library code.